tests/conftest.py

```diff
@@ -31,3 +43,24 @@ def pytest_collection_modifyitems(config, items):
     for item in items:
         if "slow" in item.keywords:
             item.add_marker(skip_slow)
+
+
+@pytest.fixture(scope="session")
+def client():
+    """Return a test instance of the flask app."""
+    with app.app_context():
+        load_test_data()
+        with app.test_client() as client:
+            yield client
+
+
+@pytest.fixture()
+def subscription():
+    """Return a single subscription object from the test dataset."""
+    with app.app_context():
+        # cisagov Libraries
+        from api.manager import SubscriptionManager
```

Comment:
Is there a benefit to importing `SubscriptionManager` down here, as opposed to importing it above with everything else?  I typically prefer to see all imports grouped together at the top of a file.

Reply:
resolved here https://github.com/cisagov/con-pca-api/pull/632/commits/890327965f766aecb99dc75f6642a086415da1bb